src/main/java/lotto/model/Lotto.java

```diff
@@ -4,9 +4,8 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-import static lotto.constant.Config.LOTTO_SIZE;
-
 public class Lotto {
+    private static final int LOTTO_SIZE = 6;
     private final List<LottoNumber> lottoNumber;
```

Comment:
```suggestion
    private static final int LOTTO_SIZE = 6;
    
    private final List<LottoNumber> lottoNumber;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다!